beast/observationmodel/ast/make_ast_input_list.py

```diff
@@ -215,7 +215,12 @@ def pick_models_toothpick_style(
             print(bin_count)
 
     # Gather the selected model seds in a table
-    sedsMags = Table(sedsMags[chosen_idxs, :], names=filters)
+    sedsMags_table = {}
+    for k, filter_name in enumerate(filters):
+        sedsMags_table[filter_name] = sedsMags[chosen_idxs, k]
+    sedsMags_table["sedgrid_indx"] = chosen_idxs
```

Comment:
sedgrid_indx has been saved in the *ASTparams.fits. You don't need to save the sedgrid_indx in an ascii file if it's unnecessary.